core/storage/job/gae_models.py

```diff
@@ -86,6 +93,17 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets at most limit jobs with respect to a time after recency_msec.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+            recency_msec: int. A time in milliseconds. After this time a
+                job is considered recent.
+
+        Returns:
+            list(Jobs) or None. A list of at most limit jobs
```

Comment:
The thing within the (...) should be the type of the object within the list -- e.g. a class name. If you need to verify you can try to find a way to trigger this function in production and do something like `logging.info(type(return_value))`.

Ditto below.